blocks/melody/ff_module/ff_module-task-event-status/ff_module-task-event-status.less

```diff
@@ -1,19 +1,26 @@
+@base-height: 1.25rem;
+@visible: 1;
+@invisible: 0.01;
+
 .ff_module-task-event-status {
 
+    font-size: .8rem;
```

Comment:
If we were using the vars in a really consistent way across the designs such that changing one at the base of the theme changed them consistently everywhere, then yes, but we're not really doing that so I just went for simplicity. Nonetheless, if you'd prefer it, no prob.